tools/cleanup/README.md

```diff
@@ -0,0 +1,6 @@
+# Get Environment Variable for Samples
```

Comment:
I don't think this belongs in the general purpose `tools` folder. It isn't relevant to many SDKs or services. In fact, it only contains a copy of a cmake module for customers to use, which we might remove, all together.

We should put this in the keyvault-secrets/samples folder, instead. Here:
https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/keyvault/azure-security-keyvault-secrets/samples

Reply:
it's not specific to secrets, , it handles keys and certificates , it is a tool to cleanup a keyvault.

Reply:
@gearama, can this go under `/keyvault/tools/cleanup` then?

Reply:
Gotcha @gearama. In that case, please put it under `keyvault/samples`.

We should keep code and files specific to a particular service under it's parent folder.
Also, sample fits better than tool in this case, using our SDK APIs.

I have seen precedence of this in another language repo:
https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/keyvault/samples

Reply:
it's a tool thus goes under tools , it's not intended to be a sample, it's useful when diagnosing tests that attempt to create a new resource, not just a new version of the resource, since our tests do not do any cleanup, relying on the fact that new resources are deployed each time the pipeline runs and are removed up once the tests are done.